apollo-client/src/main/java/com/ctrip/framework/apollo/internals/DefaultConfigManager.java

```diff
@@ -79,4 +95,8 @@ public ConfigFile getConfigFile(String namespace, ConfigFileFormat configFileFor
 
     return configFile;
   }
+
+  private String buildLockKey(String factor) {
+    return (String.format("%s.%s", getClass().getName(), factor)).intern();
```

Comment:
If we change the lock logic above, then I think intern is not necessary.

```suggestion
    return String.format("%s.%s", getClass().getName(), factor);
```

Reply:
Great suggestion, and I have been inspired, We can directly lock String to simplify it.